rent_gpu/requestor/api/templates/offer-details.html

```diff
@@ -37,6 +37,136 @@ <h2>Provider status</h2>
         {% elif offer.package == 'text-webui' %}
         Use this link to connect with Text generation WebUI: <a href="http://{{request.url.hostname}}:{{ offer.port }}" target="_blank">http://{{request.url.hostname}}:{{ offer.port }}</a>
         <br/>
+
+        <style>
+            .accordion {
+              background-color: #eee;
+              color: #444;
+              cursor: pointer;
+              padding: 10px;
+              width: 100%;
+              border: none;
+              text-align: left;
```

Comment:
For this text alignment Bootstrap has `text-start` class. https://getbootstrap.com/docs/5.0/utilities/text/

Reply:
I will change that.

Reply:
Changed